react/AvailabilitySubscriber.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useState } from 'react'
+import type { FormEvent, ChangeEvent } from 'react'
+import { useIntl } from 'react-intl'
+import { useMutation } from 'react-apollo'
+import { Button, Input } from 'vtex.styleguide'
+import useProduct from 'vtex.product-context/useProduct'
```

Comment:
Update this import to:

```suggestion
import { useProduct } from 'vtex.product-context'
```

since this was changed in the actual types exported by `vtex.product-context`. Doing it like this right now will make it easier to migrate the whole repository to use the correct types later :)